common/tracing/src/logging.rs

```diff
@@ -74,12 +73,8 @@ pub fn init_global_tracing(app_name: &str, dir: &str, level: &str) -> Vec<Worker
     // Enable log compatible layer to convert log record to tracing span.
     LogTracer::init().expect("log tracer must be valid");
 
-    // Stdout layer.
-    let (stdout_writer, stdout_guard) = tracing_appender::non_blocking(std::io::stdout());
```

Comment:
The default layer is stdout, seems no need set it here, if we set it, the stdout will be duplicated.

Reply:
query_logger is only enabled by session_mgr, by removing the stdout logger, will there be some log missing?:
https://github.com/datafuselabs/databend/blob/0fe675e92b3d8cd1d1aefab46b1729a08dbf0aa6/query/src/sessions/session_mgr.rs#L103-L108

Reply:
Sorry I was speaking about another thing...

But what is the default layer?

Reply:
From my test, it's stdout

Reply:
The query log works well:
![image](https://user-images.githubusercontent.com/172204/171774197-3eb22aa9-3a9a-465e-bcbd-89cb0a291626.png)